docs/node-operators/getting-started.mdx

```diff
@@ -25,54 +25,61 @@ This documentation is for the **Mainnet** release.
 
 ## Requirements
 
-**Software**: macOS, Linux (currently supports Debian 9 and Ubuntu 18.04 LTS), or any host with Docker
+**Software**: Supported environments include macOS, Linux (Debian 10, 11, 12 and Ubuntu 20.04 LTS), and any host machine with Docker. 
 
-:::note
-
-Windows is not officially supported at this time. However, it may be possible to use the [Docker instructions](./#docker). If you are interested in receiving a grant to develop Mina for Windows, please email grants@o1labs.org or reach out in the [Mina Protocol Discord server](https://discord.gg/minaprotocol).
+**Processor**: Only AMD64 CPU architecture is supported.
 
-:::
+**Hardware**: Sending and receiving MINA does not require any special hardware. 
 
-**Hardware**: Sending and receiving mina does not require any special hardware, but running a block producer on the Mina network currently requires:
+Running a block producer on the Mina network requires at least:
 
-- at least a 8-core processor
-- at least 16GB of RAM
+- 8-core processor
+- 16 GB of RAM
 
-Note that if you plan on running a snark worker node at the same time as a block producer, you may need more RAM. GPUs aren't currently required, but may be required for node operators when the protocol is upgraded.
+More RAM is required to run a SNARK worker node at the same time as a block producer.
```

Comment:
did you know we have a https://github.com/o1-labs/docs2/wiki/Word-list that is part of our content guidance that governs the terminology, voice and tone, and [style guidelines](https://github.com/o1-labs/docs2/wiki/docs-style-guide).